stl/inc/xstring

```diff
@@ -4478,6 +4492,20 @@ public:
     }
 #endif // _HAS_CXX20
 
+#if _HAS_CXX23
+    _NODISCARD _CONSTEXPR20_CONTAINER bool contains(const basic_string_view<_Elem, _Traits> _Right) const noexcept {
+        return basic_string_view<_Elem, _Traits>(_Mypair._Myval2._Myptr(), _Mypair._Myval2._Mysize).contains(_Right);

```

Comment:
Nitpick, I believe we conventionally use `{}` to disambiguate function calls

Reply:
Good catch - yep, that's the preferred convention in product code (although not all existing code follows it, especially for multiple-argument temporaries). @WimLeflere was following the precedent in `starts_with`/`ends_with` above which is definitely the right thing to do (local consistency is most important), but we can go ahead and change all of the occurrences in this file (more invasive cleanups should be kept separate from feature work but I think this is limited enough to be fine).